releasewarrior/commands.py

```diff
@@ -206,7 +207,8 @@ def generate_data(self):
         logger.info("updating with custom update data")
         for key_change, key_value in self.changes.items():
             if key_change == "issues":
-                new_data['builds'][-1]["issues"].extend(key_value)
+                owned_key_value = ["<{}>:{}".format(getpass.getuser(), k) for k in key_value]
```

Comment:
Yeah, you're absolutely right. I'm thinking we might hit some mismatches there or potentially no value as indicated per https://docs.python.org/2/library/getpass.html#getpass.getuser but at least it's better than nothing :)